CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ if(NOT CMAKE_BUILD_TYPE)
       "Choose the type of build from: Debug Release RelWithDebInfo MinSizeRel Coverage."
     FORCE)
 endif()
+#cmake_policy(SET CMP0063 NEW)
```

Comment:
temporary commented out?

Reply:
We are checking the problem with caffe2 build

Reply:
We saw test failures in CI. I comment this out to see if it's my PR that introduced the failure or it's master. Looks like it's in master. :(